tools/hack/create-cluster.sh

```diff
@@ -44,7 +52,38 @@ ${KIND_CFG}
 EOF
 fi
 fi
-
+if [ "$CUSTOM_CNI" = "true" ]; then
```

Comment:
Nit, non-blocking: Calico also works for "normal" e2e tests?  We can simply remove the `CUSTOM_CNI` falg and install it by default?

Reply:
For standard e2e tests, the default kindnet CNI is typically sufficient and offers a simpler, more resource-efficient setup. Introducing Calico can be reserved for scenarios that specifically require its advanced networking capabilities. 
At this point the Calico deployment adds an extra 2-3 min for each create-cluster.

Reply:
Yeah, an extra 2-3 minutes will slow down the tests, which already take a long time to finish.